src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -169,4 +171,12 @@ public double getDefaultAlleleFrequency() {
     @Argument(fullName = GET_AF_FROM_AD_LONG_NAME, doc="Use allelic depth to calculate tumor allele fraction; recommended for mitochondrial applications", optional = true)
     public boolean calculateAFfromAD = false;
 
+    /**
+     * If set to true, count overlapping read pair as two seprate reads instead of one for {@link StrandArtifact} and {@link StrandBiasBySample}
+     * This would break the independence assumption of reads and over-count the alt depth in these annotations. On the other hand it could
```

Comment:
This comment can be more direct about the fact that it is, in fact, the objectively correct behavior when it comes to strand artifacts.

Reply:
done

Reply:
done